lib/scrabble.rb

```diff
@@ -10,4 +11,48 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  #
+  def score(letter_string)
+    # letter_string.upcase! #change letter_string to upcase
+    #need to return value from point_values hash!
+    letter_string.map do |letter|
+      @point_values[letter]
+      letter.upcase!
+      letter.to_i
+      #number needs to be i, not string
+
+    end
```

Comment:
Think the main piece here is that you're mapping over the values, but then not saving the result to anything. I'm guessing that `.upcase` and `.to_i` were put in after not getting the result you expected from the map without them?